response_operations_ui/views/collection_exercise.py

```diff
@@ -405,6 +413,79 @@ def create_collection_exercise(survey_ref, short_name):
                                 new_period=form.get('period')))
 
 
+@collection_exercise_bp.route('/<short_name>/<period>/<ce_id>/create-event/<tag>', methods=['GET'])
+@login_required
+def get_create_collection_event_form(short_name, period, ce_id, tag):
+    logger.info("Retrieving form for create collection exercise event", short_name=short_name, period=period,
```

Comment:
Maybe using wrap logger from structlog would make sense here?

Reply:
As far as I am aware this is using wrap logger set up above. Or do you mean bind the attributes to the logger?